README.md

```diff
@@ -379,6 +379,72 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
 }];
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->

Reply:
?? is this a bot?

Reply:
Yes, feel free to resolve it if no need to update CHANGELOG.md